docs/hsi.md

```diff
@@ -436,15 +436,16 @@ To meet HSI-2 on systems that run this test, the result must be `valid`. *[v1.5.
 
 - [Linux Kernel TPM Documentation](https://www.kernel.org/doc/html/latest/security/tpm/tpm_event_log.html)
 
-<a id="org.fwupd.hsi.AcpiDmar"></a>
+<a id="org.fwupd.hsi.PrebootDma"></a>
```

Comment:
I think that we need to have both, i.e. two <a> sections with the old name too.

Reply:
fixed